core/modules/user/user.install

```diff
@@ -162,6 +169,18 @@ function user_schema() {
   return $schema;
 }
 
+/**
+ * Implements hook_schema_alter().
+ */
+function user_schema_alter(&$schema) {
+  // Prevent fatal error if someone clears cache before running the update.
+  // @see https://github.com/backdrop/backdrop-issues/issues/5944#issuecomment-1416900861
+  if (backdrop_get_installed_schema_version('user') < 1025) {
+    unset($schema['users']['fields']['changed']);
+    watchdog('user', 'The user module has database schema updates to install. You should run the database update script immediately.', WATCHDOG_WARNING);
```

Comment:
@herbdool - one more problem. The third argument should be an array, the the fourth the constant. Fatal error otherwise (when viewing the dblog)

Reply:
Oops. Let's see if that's better now.